custom_components/babybuddy/services.yaml

```diff
@@ -21,108 +21,53 @@ services_children_add:
       selector:
         text:
 
-services_delete:
-  name: Delete Entry
-  description: Deletes a Baby Buddy entry
+start_timer:
```

Comment:
how is timer supposed to be stopped?

Reply:
The Babybuddy api doesn't include a request to stop a timer. However, the idea is that when a service like `add_sleep` or `add_feeding` is called with the timer id in the rquest then the start/end times will be based on the timer and the timer will be automatically stopped. This is based on the Babybuddy api documentation.